src/engine/init.c

```diff
@@ -85,6 +85,8 @@ hwloc_obj_t		numa_obj;
 int			dss_num_cores_numa_node;
 /** Module facility bitmask */
 static uint64_t		dss_mod_facs;
+/** Number of storage tiers: 2 for SCM and NVMe */
+int dss_storage_tiers = 2;
```

Comment:
I'm not quite sure how this "dss_storage_tiers" will be used by io engine, and I didn't see where it's used in current patch.

Could you elaborate how it will be used?

Reply:
Right, it's not being used atm - only a preparation for further work, otherwise it may be removed.
This was carried over from my fork w/ tiering implementation in the engine. The value will be passed to bio_xsctxt_alloc(), in order to initialize the blobstore ctx for all tiers.

The changes in question are available for review (look for 'dss_nvme_tiers'): https://github.com/llasek-intc/daos/tree/llasek/tiering

Reply:
Ok, thanks.